lumin/app.package

```diff
@@ -1,5 +1,6 @@
 DATAS = "digest.sha512.signed" : "." \
         "bin/" : "bin/" \
+        "res/" : "res/" \
```

Comment:
This icon will be used internally by Lumin or by every platform?

Reply:
This is 3d icon (PlatformIcon) so we cannot reuse it in iOS/Android. But we may provide default icons for react-native as well.

Reply:
This is portal icon (= 3d icon) so only by Lumin.